packages/plugin-vue-jsx/index.js

```diff
@@ -71,8 +78,22 @@ function vueJsxPlugin(options = {}) {
     },
 
     transform(code, id, ssr) {
-      if (/\.[jt]sx$/.test(id)) {
-        const plugins = [importMeta, [jsx, options]]
+      /**
+       * @type Array<string | RegExp>
+       */
+      const defaultInclude = ['**/*.jsx', '**/*.tsx']
+      const { include, exclude, ...babelPluginOptions } = options
+
+      if (Array.isArray(include)) {
+        defaultInclude.push(...include)
+      } else if (include) {
+        defaultInclude.push(include)
+      }
```

Comment:
I think it's better to apply default when there is no value provided, instead of incremental adding to the default. See

https://github.com/vitejs/vite/blob/f4998c05cc64e4cd4114973630ce8e8503cdc38c/packages/plugin-vue/src/index.ts#L64-L67

Reply:
Emm..., I think you are right, I thought too much, I will update this later

Reply:
Already updated, please review it.